include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -4592,6 +4592,70 @@
         "extensions" : [ "SPV_KHR_ray_query" ],
         "version" : "None"
     },
+    {
+      "opname" : "OpImageSampleWeightedQCOM",
+      "class"  : "Image",
+      "opcode" : 4480,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "'Texture'" },
+        { "kind" : "IdRef", "name" : "'Coordinates'" },
+        { "kind" : "IdRef", "name" : "'Weights'" }
+      ],
+      "capabilities" : [ "TextureSampleWeightedQCOM" ],
+      "extensions" : [ "SPV_QCOM_image_processing" ],
```

Comment:
Please don't include extensions if the capability is enabled by that extensions.